docs/code_ref/coordinates/rotatedsunframe.rst

```diff
@@ -245,151 +246,27 @@ If we create a ``RotatedSun*`` frame for a different base time, we can represent
-   rs_hgs = RotatedSunFrame(base=HeliographicStonyhurst(obstime=m.date), duration=27*u.day)
-   overlay = ax.get_coords_overlay(rs_hgs)
-   overlay[0].set_ticks(spacing=15. * u.deg)
-   overlay[1].set_ticks(spacing=90. * u.deg)
-   overlay.grid(ls='-', color='blue')
-
-   plt.show()
-
-Be aware that the distorted ``RotatedSunHeliographicStonyhurst`` longitude lines are plotted on the original coordinate frame, and thus use the ``Helioprojective`` coordinate frame for the location of AIA at the original observation time (as opposed to 27 days later).
-
-Differentially rotating a Map
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. note::
-   This example requires `reproject` 0.6 or later to be installed.
-
-Here is an example of reprojecting a map to another time with an observer at a different location.
-To achieve the reprojection, one needs to transform from the original frame to the ``RotatedSun*`` version of the output frame with ``rotated_time`` set to the time of the original frame.
-When reprojecting across a change in time, it is generally advisable to use the `~sunpy.coordinates.transform_with_sun_center` context manager.
-
-.. plot::
-   :include-source:
-
-   from astropy.coordinates import SkyCoord
-   import astropy.units as u
-   from astropy.wcs import WCS
-   import matplotlib.pyplot as plt
-   from reproject import reproject_interp
-
-   from sunpy.coordinates import Helioprojective, RotatedSunFrame, HeliographicStonyhurst, transform_with_sun_center
-   from sunpy.data.sample import AIA_171_IMAGE
-   from sunpy.map import Map, make_fitswcs_header
-
-   # Load the map and define the output time with +5 days of differential rotation
-   m = Map(AIA_171_IMAGE)
-   in_time = m.date
-   out_time = in_time + 5*u.day
-
-   # Define the output real frame with an associated new observer (at Earth)
-   out_frame = Helioprojective(observer='earth', obstime=out_time)
-
-   # Define the output RotatedSunFrame (its `duration` will be -5 days)
-   rot_frame = RotatedSunFrame(base=out_frame, rotated_time=in_time)
-
-   # Construct a WCS object for the output map with the output RotatedSunFrame specified
-   out_shape = m.data.shape
-   out_center = SkyCoord(0*u.arcsec, 0*u.arcsec, frame=out_frame)
-   header = make_fitswcs_header(out_shape, out_center, scale=u.Quantity(m.scale))
-   out_wcs = WCS(header)
-   out_wcs.coordinate_frame = rot_frame
-
-   # Reproject the map from the input frame to the output RotatedSunFrame while following Sun center
-   with transform_with_sun_center():
-       arr, _ = reproject_interp(m, out_wcs, out_shape)
-
-   # Create the output map and preserve the original map's plot settings
-   out_warp = Map(arr, out_wcs)
-   out_warp.plot_settings = m.plot_settings
+* :ref:`sphx_glr_generated_gallery_differential_rotation_differentially_rotated_coordinate.py`
 
-   # Plot the output map next to the original map
-   fig = plt.figure(figsize=(12, 4))
+  .. plot:: ../examples/differential_rotation/differentially_rotated_coordinate.py
```

Comment:
Doing this provides a link to the source code, which is :+1: but I don't think it links to the right place in the built documentation. I am not sure the gallery examples are copied into built docs at their original path?

Perhaps it would be better to make this feature of sphinx gallery work: https://sphinx-gallery.github.io/stable/configuration.html#add-mini-galleries-for-api-documentation

Reply:
The source-code link works for me locally (i.e., the `.py` files are copied over to `_build/examples/...`), but I'm not using tox.  I'm guessing our tox or CI setup doesn't do the copying over.

I'll play with mini-galleries.

Reply:
I tweaked the CircleCI configuration to copy over the missing artifacts, so now the links work (see [recent build](https://48356-2165383-gh.circle-artifacts.com/0/docs/_build/html/code_ref/coordinates/rotatedsunframe.html))

Mini-galleries look like they'd be neat, but I can't seem to get them to work...

Reply:
Okay, I finally got mini-galleries to work, but I'm not a fan of how it ends up looking here.  I can't change the thumbnail size independently of the main gallery, and the heading text and level are not configurable.

I'm inclined to stick with the `plot` directive.

Reply:
On further thought, even though I fixed it on CircleCI, it just occurred to me that this probably won't work on RTD?  Maybe it has to be changed to a mini-gallery...

Reply:
It is now a mini-gallery.  I realized that I could clip out the problematic heading in the included file.